server/controllers/ArticleController.js

```diff
@@ -0,0 +1,252 @@
+import slugCreate from 'slug';
+import uuid from 'uuid/v1';
+import models from '../database/models';
+
+const { Article, User } = models;
+
+/**
+ *
+ * The ArticleController contains static methods that are used as
+ * controllers to handle the different article routes.
+ *  @class ArticleController
+ *  @returns {undefined} this is a class thus does not return anything
+ */
+class ArticleController {
+  /**
+   * Method adds a new article to the database
+   * It returns a json response of the article created
+   * It handles the POST /api/articles.
+   * @param {Object} req the request body
+   * @param {Object} res the response body
+   * @param {function} next a call to the next function
+   * @returns {Object} the response body
+   */
+  static createArticle(req, res, next) {
+    const {
+      title, description, body,
+    } = req.body.article;
```

Comment:
If I make a request to create an article without an article in the body, it will through an error, you need to check if `article` is in the body. You will need to check this for all the routes that receive an article from the body as well.

Reply:
my validation already cover that

Reply:
done

Reply:
done

Reply:
done through validation

Reply:
Noted